Android/app/src/main/java/app/intra/net/socks/TLSProbe.java

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
Maybe this class should live on a `tls` package, rather than `socks`?

Reply:
The `socks` package contains all the code that is only used in SOCKS mode.  For example, it also contains `LocalhostResolver`, which is a generic DNS-to-DoH forwarder.

I'd rather not add a new package that only contains one class.